lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -307,13 +311,11 @@ def verify_file(self, path):
         Verify plugin configuration file and mark this plugin active
         Args:
             path: Path of configuration YAML file
-
         Returns: True if everything is correct, else False
-
         """
         valid = False
         if super(InventoryModule, self).verify_file(path):
-            if path.endswith(('vmware.yaml', 'vmware.yml')):
+            if path.endswith(('vmware.yaml', 'vmware.yml', 'vmware_vm_inventory.yaml', 'vmware_vm_inventory.yml')):
```

Comment:
If `verify_file` is required, I've updated to allow `vmware_vm_inventory.yml|yaml` to be consistent with naming convention of other inventory plugins